collectors/APICalls/breakpoints.js

```diff
@@ -32,15 +32,15 @@ const breakpoints = [
             }
         ]
     },
-    {
-        global: 'chrome',
-        props: [
-        ],
-        methods: [
-            // {name: 'csi'}, // doesn't work
-            // {name: 'loadTimes'}, // doesn't work
-        ]
-    },
+    // {
+    //     global: 'chrome',
+    //     props: [
+    //     ],
+    //     methods: [
+    //         // {name: 'csi'}, // doesn't work
+    //         // {name: 'loadTimes'}, // doesn't work
+    //     ]
+    // },
```

Comment:
I commented out all sections that don't have any methods or properties to track.

Reply:
These were just placeholders?

Reply:
Yeah, pretty much. This file is a bit messy because there are many APIs here that are commented out because we can't track them using existing method (because either they are static values or don't exist in headless Chromium). I'm keeping them around as notes, but I realize it's messy - we should move them to docs.